clients/cli/src/main/java/org/apache/gravitino/cli/commands/DeleteRole.java

```diff
@@ -38,28 +39,30 @@ public class DeleteRole extends Command {
    * @param ignoreVersions If true don't check the client/server versions match.
    * @param force Force operation.
    * @param metalake The name of the metalake.
-   * @param role The name of the role.
+   * @param roles The name of the role.
    */
   public DeleteRole(
-      String url, boolean ignoreVersions, boolean force, String metalake, String role) {
+      String url, boolean ignoreVersions, boolean force, String metalake, String[] roles) {
     super(url, ignoreVersions);
     this.metalake = metalake;
     this.force = force;
-    this.role = role;
+    this.roles = roles;
   }
 
   /** Delete a role. */
   @Override
   public void handle() {
-    boolean deleted = false;
+    boolean deleted = true;
```

Comment:
I see why you changed this, but it seems a bit misleading, as we've not yet deleted anything.